src/main/java/cropcert/certification/ApiConstants.java

```diff
@@ -4,6 +4,10 @@
 package cropcert.certification;
 
 public class ApiConstants {
+	private ApiConstants() {
+
```

Comment:
include super() keyword here for best practice.Look into observation module for reference.